app/assets/stylesheets/profile.scss

```diff
@@ -75,6 +73,10 @@
         list-style: none;
         margin: 0;
         padding: 0;
+        display: inline-block;
+        background-color: #fff;
+        border-radius: 3px;
+        border: 1px solid #dadada;
```

Comment:
Color literals like `#dadada` should only be used in variable declarations; they should be referred to via variable everywhere else.